content/tutorials/spatial_standards_vector/index.Rmd

```diff
@@ -0,0 +1,202 @@
+
+
+## How to make and use GeoJSON files (`*.geojson`)
+
+### Making a GeoJSON file from a geospatial `sf` object in R
+
+As another example, let's download a shapefile of stream habitat 3260 in Flanders (version _2018_):
+
+```{r}
+download_zenodo(doi = "10.5281/zenodo.3386246")
+```
+
+The data source is a shapefile again, in this case consisting of 4 different files.
+Similar as above, we read the geospatial data into R as an `sf` object and select a few attributes to work with:
+
+```{r}
+habitatstreams <- 
+  read_sf("habitatstreams.shp") %>% 
+  select(river_name = NAAM,
+         source = BRON)
+```
+
+
+```{r paged.print=FALSE}
+habitatstreams
+```
+
+Nowadays, it is recommended to use the more recent and strict **RFC7946** implementation of GeoJSON.
+The previous 'GeoJSON 2008' implementation is now obsoleted
+(see [the post](../../articles/geospatial_standards/) on this tutorials website for a bit more background).
+
+The RFC7946 standard is well supported by GDAL's GeoJSON driver, however GDAL must be given the explicit option `RFC7946=YES` in order to use it already ^[
+Though GeoJSON 2008 is [obsoleted](http://geojson.org/geojson-spec.html), the now recommended [RFC7946](https://tools.ietf.org/html/rfc7946) standard is still officially in a _proposal_ stage.
+That is probably the reason why GDAL does not yet default to RFC7946.
+A somehow confusing stage, it seems.
+].
+
+Write the GeoJSON file as follows:
+
+```{r}
+habitatstreams %>% 
+  st_write("habitatstreams.geojson",
+           layer_options = "RFC7946=YES")
+```
+
+Done creating!
+
+### Do I look good?
+
+Hey wait, wasn't a GeoJSON file just a text file?\
+_**Indeed.**_
+
+So I can just open it as a text file to get an idea of its contents?\
+_**Well seen :-)**_
+
+Hence, also use it in versioned workflows?\
+_**Didn't hear that. (Cool, though...)**_
+
+Let's just look at the top 7 lines of the file:
+
+```{r echo=FALSE, comment=""}
+readLines("habitatstreams.geojson", n = 7) %>% 
+  cat(sep = "\n")
+```
+
+You can see it basically lists the feature attributes and the coordinates of the lines' vertices, with each feature starting on a new line.
+
+Compare the coordinates with those of the `sf` object `habitatstreams` above: the data have been reprojected on the fly to WGS84!
+
+Note: in order to be still manageable (text file size, usage in versioning systems) it seems wise to use GeoJSON for more simple cases -- **points and rather simple lines and polygons** -- and use the binary GeoPackage format for larger (more complex) cases.
+
+### Reading a GeoJSON file
+
+Just do this:
+
+```{r}
+habitatstreams_test <- st_read("habitatstreams.geojson")
+```
+
+Same story as for the GeoPackage: other geospatial software will (or should be) able to open the GeoJSON format as well, as it's an open and well established standard.
+
+From the message of `st_read()` you can see the CRS is WGS84 ([EPSG-code 4326](https://epsg.io/4326)) - this is always expected when reading a GeoJSON file.
```

Comment:
This is another difference between both formats that can be added to a separate section that compares both and recommends when to use which.

Reply:
The recommendation to make a comparative section sounds good. Will try.

Reply:
Handled by 890e8a8